### DISCUSSION TITLE
Why does the page do not deploy?

### DISCUSSION THREAD

**Discussion by glsasan** [community/community#48159] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Newbie in github here.

Why does my page look like this when I deploy?

https://glsasan.github.io/iamwebdev.github.io/

I have all the files in the repository.

**Comment by radar-blip** [comment]
There is only a README file in your repository, so that is what you are seeing.

**Reply by glsasan** [reply]
There is an index.html and style.css under my repository and a couple of folders for js, images and videos. I tried deleting the readme.md and it gave me a 404 error.

**Comment by melvincwng** [comment]
@glsasan, to host your own static website on GitHub Pages, you need to ensure the naming of your repository is correct.

For starters, you need to change your repository's name to **glsasan.github.io**. This would unlock a special repository that allows us to host your own static website.

Next, deploy your static website again using GitHub Pages.

Once deployed, your website should be live at: https://glsasan.github.io/

If you need additional reference, feel free to follow steps 1 to 5 in this GitHub Pages Documentation link over [here](https://pages.github.com/).

Hope this clarifies and answers your question 😃
